src/utilities/allNotes-utils.jsx

```diff
@@ -0,0 +1,71 @@
+import axios from "axios";
+
+import { getToken } from "./helper-utils";
+
+const getAllNotes = async () => {
+  try {
+    const { data, status } = await axios.get("/api/notes", {
+      headers: {
+        authorization: getToken(),
+      },
+    });
+    if (status !== 200) return;
+
+    return data.notes;
+  } catch (err) {
+    console.log(err.response);
+  }
+};
+
+const addNote = async (note, setNotes) => {
+  try {
+    const { data, status } = await axios.post(
+      "/api/notes",
+      { note },
+      {
+        headers: {
+          authorization: getToken(),
+        },
+      }
+    );
+    if (status !== 201) return;
+    setNotes(data.notes);
+  } catch (err) {
+    console.log(err.response);
+  }
+};
+
+const editNote = async (id, note, setNotes) => {
+  console.log(id);
+  try {
+    const { data, status } = await axios.post(
+      `/api/notes/${id}`,
```

Comment:
You can create a seprate file for all api calls and give the variable name to api so that you don't make spelling mistake while calling api